esrally/driver/driver.py

```diff
@@ -71,18 +71,25 @@ class StartBenchmark:
     pass
 
 
+class RallyConfig:
+    """
+    Prompts loading of a track
```

Comment:
It's a bit unclear to me what this is actually doing, by just reading the docstring. I fail to understand how an object instantiating this that stores `cfg` prompts (as in causes/brings about?) the loading of a track. Is there anything short we could add here to make this easier to understand in the future without having to git blame and reference the issue https://github.com/elastic/rally/issues/1206?

Reply:
Great point. I think I will go the "self-documenting" route here.  Other message classes are verbs, telling the actor what to do, this one would be better named "Bootstrap" and could in the future denote other activity actors should be doing before their intended work.